src/notes/components/Sidebar.tsx

```diff
@@ -34,15 +46,37 @@ const Sidebar = ({
         minWidth: width,
       }}
     >
-      <SidebarNotes
-        notes={notes}
-        active={active}
-        onActivateNote={onActivateNote}
-        onNoteContextMenu={onNoteContextMenu}
-        canChangeOrder={canChangeOrder}
-        onChangeOrder={onChangeOrder}
-        openNoteOnMouseHover={openNoteOnMouseHover}
-      />
+      <div id="sidebar-notes-container">
+        {pinnedNotes.length > 0 && (
+          <SidebarNotes
+            id="sidebar-notes-pinned"
+            notes={pinnedNotes}
+            {...commonSidebarNotesProps}
+            canDragEnter={onDraggingNoteOriginator === "sidebar-notes-pinned"}
+            onChangeOrder={(newOrder) => onChangeOrder([
+              ...newOrder,
+              ...unpinnedNotes.map((note) => note.name),
+            ])}
+          />
+        )}
+
+        {pinnedNotes.length > 0 && unpinnedNotes.length > 0 && (
+          <div id="sidebar-notes-separator" />
+        )}
```

Comment:
There's **_one empty line_** between pinned and unpinned notes so they can be seen as two separate groups.